core/src/main/kotlin/ch/ergon/dope/resolvable/clause/model/ReturningClause.kt

```diff
@@ -30,3 +32,27 @@ class ReturningClause(
         )
     }
 }
+
+class UpdateReturningClause(
+    private val field: Field<out ValidType>,
+    private vararg val fields: Field<out ValidType>,
+    private val parentClause: IUpdateLimitClause,
+) : IUpdateReturningClause {
+
+    override fun toDopeQuery(): DopeQuery {
+        val fieldsDopeQuery = fields.map { it.toDopeQuery() }
+        val fieldDopeQuery = field.toDopeQuery()
+        val parentDopeQuery = parentClause.toDopeQuery()
+        return DopeQuery(
+            queryString = formatToQueryStringWithSymbol(
+                parentDopeQuery.queryString,
+                "RETURNING",
```

Comment:
please extract to a constant

Reply:
I extracted the logic of both `...ReturningClause`s into their own sealed class, analogous to other clauses to prevent code/logic duplication